### PR TITLE
[Merged by Bors] - chore: adaptations for nightly-2024-07-31

### PR DESCRIPTION
**Pull request by kim-em**

### CODE REVIEW

File: Mathlib/Data/Fin/Basic.lean [hunk 1]
```diff
@@ -191,6 +191,8 @@ section Order
 ### order
 -/
 
+
```
Comment (Ruben-VandeVelde) [inline Mathlib/Data/Fin/Basic.lean:194]: (These whitespace changes make bump/v4.11.0 match master, so should be kept)

**Review by Ruben-VandeVelde — APPROVED** [review]
Thanks!

maintainer merge

**Comment by fpvandoorn** [comment]
bors merge

**Review by Vierkantor — APPROVED** [review]
LGTM!

bors r+

**Pull request closed without merging**